### ISSUE TITLE
Question about the results of surface normal

### ISSUE THREAD

**Issue by JYChen18** [Shreeyak/cleargrasp#27]
Hi Shreeyak,
I want to repeat the number reported in your paper and I have run the experiment of surface normal using the same setting as in config.yaml.sample. I use your code and dataset without any modification. And I simply use one GPU with batchsize=8. To my surprise, only after one epoch, the mean/median error on test synthetic dataset becomes 18.85/10.63, which is much better than the number reported in your paper 25.32 / 20.53. I think I must missing something but I just can't find it. Do you have any idea about that?

**Comment by Shreeyak** [comment]
Hi @JYChen18, 
I'm glad the code worked for you. I would trust the results of your run - it's possible for it to improve further.

I trained those models a long time ago and I don't remember the details. It's quite possible that there was some mistake in that run (bad hyperparameters?). We were surprised to see such poor performance on the synthetic data but didn't have time to go back and rerun experiments.

**Comment by hyt1004** [comment]
> Hi Shreeyak, I want to repeat the number reported in your paper and I have run the experiment of surface normal using the same setting as in config.yaml.sample. I use your code and dataset without any modification. And I simply use one GPU with batchsize=8. To my surprise, only after one epoch, the mean/median error on test synthetic dataset becomes 18.85/10.63, which is much better than the number reported in your paper 25.32 / 20.53. I think I must missing something but I just can't find it. Do you have any idea about that?


I also has this problem. That’s because the code caculates the mean/median error with all pixels. However the mean/median error was caculated only with the pixels belong transparent objects in the paper. You should input the mask directory.
